### ISSUE TITLE
 very need for demo on Segmentation Phrase Grounding

### ISSUE THREAD

**Issue by arlisgreat** [IDEA-Research/DINO-X-API#3]

**Comment by rentainhe** [comment]
Hello, you can use some open-sourced language tools to get the noun chunks from the sentence, like `spacy`, then you can concatenate them using `.` and use DINO-X to predict their corresponding boxes and masks, we will offer some examples in the future release.

**Comment by rentainhe** [comment]
There are still some errors in our prediction mask API, we will fix them later and release the segmentation demo, and we are still enhancing the capabilities of the mask head.

**Comment by arlisgreat** [comment]
@rentainhe great thanks to you!!!!! we need it for our coureswork,so when will it be released?

**Comment by rentainhe** [comment]
We will release it within this week

**Comment by xyb1314** [comment]
Waiting for prediction mask API, too! Find hard to parse the mask=ObjectMask params in the response !

**Comment by rentainhe** [comment]
> Waiting for prediction mask API, too! Find hard to parse the mask=ObjectMask params in the response !

@xyb1314 We have already supported DINO-X with SAM 2 for object segmentation and tracking in: https://github.com/IDEA-Research/Grounded-SAM-2, which may be another choice for your needs. You can have a try after registering the DINO-X token quotas.
